[2024-02-14T20:03:24.278+0000] {taskinstance.py:1957} INFO - Dependencies all met for dep_context=non-requeueable deps ti=<TaskInstance: dag_with_postgres_operator.create_table_in_pg_db scheduled__2024-02-13T00:00:00+00:00 [queued]>
[2024-02-14T20:03:24.283+0000] {taskinstance.py:1957} INFO - Dependencies all met for dep_context=requeueable deps ti=<TaskInstance: dag_with_postgres_operator.create_table_in_pg_db scheduled__2024-02-13T00:00:00+00:00 [queued]>
[2024-02-14T20:03:24.283+0000] {taskinstance.py:2171} INFO - Starting attempt 1 of 6
[2024-02-14T20:03:24.320+0000] {taskinstance.py:2192} INFO - Executing <Task(PostgresOperator): create_table_in_pg_db> on 2024-02-13 00:00:00+00:00
[2024-02-14T20:03:24.324+0000] {standard_task_runner.py:60} INFO - Started process 58615 to run task
[2024-02-14T20:03:24.327+0000] {standard_task_runner.py:87} INFO - Running: ['airflow', 'tasks', 'run', 'dag_with_postgres_operator', 'create_table_in_pg_db', 'scheduled__2024-02-13T00:00:00+00:00', '--job-id', '289', '--raw', '--subdir', 'DAGS_FOLDER/pg_op_dag.py', '--cfg-path', '/tmp/tmpaoq7_q__']
[2024-02-14T20:03:24.328+0000] {standard_task_runner.py:88} INFO - Job 289: Subtask create_table_in_pg_db
[2024-02-14T20:03:24.379+0000] {task_command.py:423} INFO - Running <TaskInstance: dag_with_postgres_operator.create_table_in_pg_db scheduled__2024-02-13T00:00:00+00:00 [running]> on host codespaces-57ffd0
[2024-02-14T20:03:24.469+0000] {taskinstance.py:2481} INFO - Exporting env vars: AIRFLOW_CTX_DAG_OWNER='raghu' AIRFLOW_CTX_DAG_ID='dag_with_postgres_operator' AIRFLOW_CTX_TASK_ID='create_table_in_pg_db' AIRFLOW_CTX_EXECUTION_DATE='2024-02-13T00:00:00+00:00' AIRFLOW_CTX_TRY_NUMBER='1' AIRFLOW_CTX_DAG_RUN_ID='scheduled__2024-02-13T00:00:00+00:00'
[2024-02-14T20:03:24.469+0000] {sql.py:276} INFO - Executing: create table if not exists dag_runs (name varchar(50), age int)
[2024-02-14T20:03:24.472+0000] {base.py:83} INFO - Using connection ID 'postgres_db_connection' for task execution.
[2024-02-14T20:03:24.493+0000] {taskinstance.py:2699} ERROR - Task failed with exception
Traceback (most recent call last):
  File "/home/codespace/.local/share/virtualenvs/airflow_sandbox-vp-HKq_L/lib/python3.10/site-packages/airflow/models/taskinstance.py", line 433, in _execute_task
    result = execute_callable(context=context, **execute_callable_kwargs)
  File "/home/codespace/.local/share/virtualenvs/airflow_sandbox-vp-HKq_L/lib/python3.10/site-packages/airflow/providers/common/sql/operators/sql.py", line 277, in execute
    hook = self.get_db_hook()
  File "/home/codespace/.local/share/virtualenvs/airflow_sandbox-vp-HKq_L/lib/python3.10/site-packages/airflow/providers/common/sql/operators/sql.py", line 188, in get_db_hook
    return self._hook
  File "/usr/local/python/3.10.13/lib/python3.10/functools.py", line 981, in __get__
    val = self.func(instance)
  File "/home/codespace/.local/share/virtualenvs/airflow_sandbox-vp-HKq_L/lib/python3.10/site-packages/airflow/providers/common/sql/operators/sql.py", line 151, in _hook
    hook = conn.get_hook(hook_params=self.hook_params)
  File "/home/codespace/.local/share/virtualenvs/airflow_sandbox-vp-HKq_L/lib/python3.10/site-packages/airflow/models/connection.py", line 365, in get_hook
    hook_class = import_string(hook.hook_class_name)
  File "/home/codespace/.local/share/virtualenvs/airflow_sandbox-vp-HKq_L/lib/python3.10/site-packages/airflow/utils/module_loading.py", line 35, in import_string
    module_path, class_name = dotted_path.rsplit(".", 1)
AttributeError: 'NoneType' object has no attribute 'rsplit'
[2024-02-14T20:03:24.495+0000] {taskinstance.py:1138} INFO - Marking task as UP_FOR_RETRY. dag_id=dag_with_postgres_operator, task_id=create_table_in_pg_db, execution_date=20240213T000000, start_date=20240214T200324, end_date=20240214T200324
[2024-02-14T20:03:24.530+0000] {standard_task_runner.py:107} ERROR - Failed to execute job 289 for task create_table_in_pg_db ('NoneType' object has no attribute 'rsplit'; 58615)
[2024-02-14T20:03:24.539+0000] {local_task_job_runner.py:234} INFO - Task exited with return code 1
[2024-02-14T20:03:24.549+0000] {taskinstance.py:3281} INFO - 0 downstream tasks scheduled from follow-on schedule check
[2024-02-14T20:17:17.698+0000] {taskinstance.py:1957} INFO - Dependencies all met for dep_context=non-requeueable deps ti=<TaskInstance: dag_with_postgres_operator.create_table_in_pg_db scheduled__2024-02-13T00:00:00+00:00 [queued]>
[2024-02-14T20:17:17.703+0000] {taskinstance.py:1957} INFO - Dependencies all met for dep_context=requeueable deps ti=<TaskInstance: dag_with_postgres_operator.create_table_in_pg_db scheduled__2024-02-13T00:00:00+00:00 [queued]>
[2024-02-14T20:17:17.703+0000] {taskinstance.py:2171} INFO - Starting attempt 1 of 6
[2024-02-14T20:17:17.742+0000] {taskinstance.py:2192} INFO - Executing <Task(PostgresOperator): create_table_in_pg_db> on 2024-02-13 00:00:00+00:00
[2024-02-14T20:17:17.745+0000] {standard_task_runner.py:60} INFO - Started process 67008 to run task
[2024-02-14T20:17:17.748+0000] {standard_task_runner.py:87} INFO - Running: ['airflow', 'tasks', 'run', 'dag_with_postgres_operator', 'create_table_in_pg_db', 'scheduled__2024-02-13T00:00:00+00:00', '--job-id', '320', '--raw', '--subdir', 'DAGS_FOLDER/pg_op_dag.py', '--cfg-path', '/tmp/tmp9ik2jh_j']
[2024-02-14T20:17:17.749+0000] {standard_task_runner.py:88} INFO - Job 320: Subtask create_table_in_pg_db
[2024-02-14T20:17:17.801+0000] {task_command.py:423} INFO - Running <TaskInstance: dag_with_postgres_operator.create_table_in_pg_db scheduled__2024-02-13T00:00:00+00:00 [running]> on host codespaces-57ffd0
[2024-02-14T20:17:17.891+0000] {taskinstance.py:2481} INFO - Exporting env vars: AIRFLOW_CTX_DAG_OWNER='raghu' AIRFLOW_CTX_DAG_ID='dag_with_postgres_operator' AIRFLOW_CTX_TASK_ID='create_table_in_pg_db' AIRFLOW_CTX_EXECUTION_DATE='2024-02-13T00:00:00+00:00' AIRFLOW_CTX_TRY_NUMBER='1' AIRFLOW_CTX_DAG_RUN_ID='scheduled__2024-02-13T00:00:00+00:00'
[2024-02-14T20:17:17.891+0000] {sql.py:276} INFO - Executing: create table if not exists dag_runs (name varchar(50), age int);
[2024-02-14T20:17:17.895+0000] {base.py:83} INFO - Using connection ID 'postgres_db_connection' for task execution.
[2024-02-14T20:17:17.915+0000] {base.py:83} INFO - Using connection ID 'postgres_db_connection' for task execution.
[2024-02-14T20:17:19.867+0000] {sql.py:450} INFO - Running statement: create table if not exists dag_runs (name varchar(50), age int);, parameters: None
[2024-02-14T20:17:20.736+0000] {taskinstance.py:1138} INFO - Marking task as SUCCESS. dag_id=dag_with_postgres_operator, task_id=create_table_in_pg_db, execution_date=20240213T000000, start_date=20240214T201717, end_date=20240214T201720
[2024-02-14T20:17:20.807+0000] {local_task_job_runner.py:234} INFO - Task exited with return code 0
[2024-02-14T20:17:20.816+0000] {taskinstance.py:3281} INFO - 0 downstream tasks scheduled from follow-on schedule check
[2024-02-14T20:20:51.931+0000] {taskinstance.py:1957} INFO - Dependencies all met for dep_context=non-requeueable deps ti=<TaskInstance: dag_with_postgres_operator.create_table_in_pg_db scheduled__2024-02-13T00:00:00+00:00 [queued]>
[2024-02-14T20:20:51.936+0000] {taskinstance.py:1957} INFO - Dependencies all met for dep_context=requeueable deps ti=<TaskInstance: dag_with_postgres_operator.create_table_in_pg_db scheduled__2024-02-13T00:00:00+00:00 [queued]>
[2024-02-14T20:20:51.936+0000] {taskinstance.py:2171} INFO - Starting attempt 1 of 6
[2024-02-14T20:20:51.974+0000] {taskinstance.py:2192} INFO - Executing <Task(PostgresOperator): create_table_in_pg_db> on 2024-02-13 00:00:00+00:00
[2024-02-14T20:20:51.977+0000] {standard_task_runner.py:60} INFO - Started process 68691 to run task
[2024-02-14T20:20:51.980+0000] {standard_task_runner.py:87} INFO - Running: ['airflow', 'tasks', 'run', 'dag_with_postgres_operator', 'create_table_in_pg_db', 'scheduled__2024-02-13T00:00:00+00:00', '--job-id', '320', '--raw', '--subdir', 'DAGS_FOLDER/pg_op_dag.py', '--cfg-path', '/tmp/tmplo9u2lun']
[2024-02-14T20:20:51.981+0000] {standard_task_runner.py:88} INFO - Job 320: Subtask create_table_in_pg_db
[2024-02-14T20:20:52.046+0000] {task_command.py:423} INFO - Running <TaskInstance: dag_with_postgres_operator.create_table_in_pg_db scheduled__2024-02-13T00:00:00+00:00 [running]> on host codespaces-57ffd0
[2024-02-14T20:20:52.137+0000] {taskinstance.py:2481} INFO - Exporting env vars: AIRFLOW_CTX_DAG_OWNER='raghu' AIRFLOW_CTX_DAG_ID='dag_with_postgres_operator' AIRFLOW_CTX_TASK_ID='create_table_in_pg_db' AIRFLOW_CTX_EXECUTION_DATE='2024-02-13T00:00:00+00:00' AIRFLOW_CTX_TRY_NUMBER='1' AIRFLOW_CTX_DAG_RUN_ID='scheduled__2024-02-13T00:00:00+00:00'
[2024-02-14T20:20:52.137+0000] {sql.py:276} INFO - Executing: create table if not exists dag_runs (name varchar(50), age int);
[2024-02-14T20:20:52.141+0000] {base.py:83} INFO - Using connection ID 'postgres_db_connection' for task execution.
[2024-02-14T20:20:52.164+0000] {base.py:83} INFO - Using connection ID 'postgres_db_connection' for task execution.
[2024-02-14T20:20:53.413+0000] {sql.py:450} INFO - Running statement: create table if not exists dag_runs (name varchar(50), age int);, parameters: None
[2024-02-14T20:20:54.119+0000] {taskinstance.py:1138} INFO - Marking task as SUCCESS. dag_id=dag_with_postgres_operator, task_id=create_table_in_pg_db, execution_date=20240213T000000, start_date=20240214T202051, end_date=20240214T202054
[2024-02-14T20:20:54.167+0000] {local_task_job_runner.py:234} INFO - Task exited with return code 0
[2024-02-14T20:20:54.181+0000] {taskinstance.py:3281} INFO - 1 downstream tasks scheduled from follow-on schedule check
